html-css/week1/bayram/order-desktop/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <title>PAYMENT FORM</title>
+  </head>
+
+  <body>
+    <div class="summary-container">
+      <div class="order-summary">Order Summary</div>
+      <div class="total-container">
+        <span class="total">Total</span>
+        <span>$333 / month</span>
+      </div>
+      <div class="separator"></div>
+      <div class="total-container total-bold">
+        <span class="total">Total</span>
+        <span>$100</span>
+      </div>
+    </div>
+
+    <div class="payment-container">
+      <h2 class="payment-title">Payment Method</h2>
+      <div class="box-container">
+        <div class="payment-box">
+          <img src="images/cards.png" alt="cards" class="payment-image" />
+          <span class="payment-text">Credit / Debit</span>
+        </div>
+        <div class="payment-box">
+          <img
+            src="images/paypal-updated.png"
+            alt="paypal"
+            class="payment-image"
+          />
+          <span class="payment-text">PayPal</span>
+        </div>
+      </div>
+    </div>
+
+    <form>
+      <div class="card-container">
+        <h2 class="card-info">Card Info</h2>
+        <div class="multi-card-container">
+          <div class="card-input-container card-number">
+            <label for="card-number">Card number</label>
+            <input
+              type="text"
+              class=""
+              placeholder="1234 1234 1234 1234"
+              name="card-number"
+              required
+            />
+            <span class="tooltip">?</span>
+          </div>
+
+          <div class="card-input-container card-expiry">
+            <label for="exp-date">Expiry date</label>
+            <input
+              type="text"
+              class=""
+              placeholder="MM / YY"
+              name="exp-date"
+              required
+            />
+          </div>
+
+          <div class="card-input-container card-security">
+            <label for="sec-code">Security code</label>
+            <input
+              type="text"
+              class=""
+              placeholder="CVV"
+              name="sec-code"
+              required
+            />
+            <span class="tooltip">?</span>
+          </div>
+        </div>
+      </div>
+
+      <button type="submit" class="buy-now" value="Buy Now">Buy Now</button>
+      <p class="help-text">Need Help?</p>
```

Comment:
This would most probably be an `<a>` tag.
```suggestion
      <a href="#" class="help-text">Need Help?</a>
```